src/dmd/common/string.d

```diff
@@ -207,3 +207,23 @@ version(Windows) auto extendedPathThen(alias F)(const(char)[] path)
     // Strip null terminator from the slice
     return F(absPath[0 .. $ - 1]);
 }
+
+/**
+ * Returns true if `s` starts with `prefix`, false otherwise.
+ */
+bool startsWith(const(char)[] s, const(char)[] prefix)
+{
+    if (s.length < prefix.length)
+        return false;
+    return s[0 .. prefix.length] == prefix;
+}
+
+/**
+ * Returns true if `s` ends with `suffix`, false otherwise.
```

Comment:
Ddoc

Reply:
http://dtest.dlang.io/artifact/website-b0500bb994fbd83fc3e6192d631db59bd84ec19a-52980191da4ba5776ce3b3738b0dc5cf/web/library-prerelease/dmd/common/string/starts_with.html

although there is a convention to be stuck to this *is* present in the generated docs